### ISSUE TITLE
Livejasmine login error, software caused connection abort: socket write error

### ISSUE THREAD

**Issue by lordmaker** [0xboobface/ctbrec#149]
Hey, i get that error warning and the livejasmine site pops up in a different windows and my account is logged in on that pop-up window but it wont show anything on the livejasmine tab in the ctb recorder, i get the follow promt/error "Livejasmine login error software caused connection abort: socket write error".

**Comment by 0xboobface** [comment]
Can you please attach a screenshot?

**Comment by lordmaker** [comment]
![screenshot_1](https://user-images.githubusercontent.com/46866270/51440688-fa9b5980-1cc9-11e9-83e8-d6e943bd1893.png)
![screenshot_2](https://user-images.githubusercontent.com/46866270/51440689-fa9b5980-1cc9-11e9-9436-b2bf89cac046.png)

**Comment by 0xboobface** [comment]
Did the login work in the popup browser window?

**Comment by lordmaker** [comment]
yeah I was able to log in in the pop up broweser but I didn’t have the ability to record anything

**Comment by 0xboobface** [comment]
Did you get it to work in the meantime? The idea is, that you login in and then the session is transfered over to ctbrec. It seems, that this step didn't work as expected. If you didn't get it to work, you can try the following steps:
* close ctbrec
* <kbd>Windows</kbd> + <kbd>R</kbd>
* Type in `%appdata%` <kbd>Enter</kbd>
* delete the ctbrec-minimal-browser directory
* open the ctbrec directory and delete cookies-livejasmin.json
* start ctbrec and try again

**Comment by lordmaker** [comment]
![screenshot_1](https://user-images.githubusercontent.com/46866270/51441714-7dc1ad00-1cd4-11e9-897c-5c3b926f1936.png)


it's stuck on logging in, nothing else happens

**Comment by 0xboobface** [comment]
If you close the browser manually, the error occurs?

**Comment by lordmaker** [comment]
yep

**Comment by 0xboobface** [comment]
Mhh, I guess the communication between ctbrec and the browser window does not work properly, but I don't have a clue why. It is working in from ctbrec to the browser, otherwise you would not see the LiveJasmin page. But the communication back does not seem to work :disappointed:

**Comment by 0xboobface** [comment]
Can you please attach the log file. It's called ctbreg.log and is located right next to the ctbrec.exe

**Comment by andyxm715** [comment]
works fine on my Windows 10 VM, I put my login details into the Settings -> livejasmin tab, the little browser page opens with my details already entered, just click on login, and can see it has logged me in and the browser window closes. I can click on the tabs on the left side which populate with the online girls and the girls in my followed list,  I can click on the online girl and records fine, been working ok since last night, I'm still testing at the moment but looks spot on for me..

**Comment by 0xboobface** [comment]
@andyxm715 Thanks for the feedback. That is how it is supposed to work.

**Comment by lordmaker** [comment]
[ctbrec.log](https://github.com/0xboobface/ctbrec/files/2776861/ctbrec.log)

**Comment by 0xboobface** [comment]
Mhh, nothing suspicious in there. It might be a timing problem. I'm gonna test a few things and get back.

**Comment by lordmaker** [comment]
Sure

**Comment by 0xboobface** [comment]
I had another look at the code, but didn't really find a possible reason.
Is it possible, that this is a firewall problem? Are you using a personal firewall, which might be blocking the communication between ctbrec and the browser popup?

**Comment by lordmaker** [comment]
What do you mean by personal firewall ?

**Comment by 0xboobface** [comment]
A firewall software, which is running on your Windows machine.

**Comment by lordmaker** [comment]
I have the standard settings that windows firewall has, I have no other specialized software that Deals with the firewall.

**Comment by 0xboobface** [comment]
OK, then I still have no clue, why this is happening :disappointed:

**Comment by Sojiro84** [comment]
Same issue for me as well. Also tried the suggested fixes with no luck. I am on Linux.

**Comment by lordmaker** [comment]
Yeah I checked my firewall to and it was operating as normal. So I have no idea what to do or how to fix it. I guess we’ll have to wait until there’s a solution to our problem.

**Comment by 0xboobface** [comment]
By now I tested it on:
* Gentoo Linux (native install)
* Windows 10 (native install)
* Windows 10 (virtual machine)
* Arch Linux (virtual machine)
* Ubuntu 18.04 (virtual machine)

It worked in all cases.

@Sojiro84 Can you install telnet (if not installed) and try the following:
* open a terminal
* cd into ctbrec/browser
* execute the browser: `./ctbrec-minimal-browser` This will open the popup browser with a blank white page
* open another terminal
* `telnet localhost 3202`
* type / copy in the following text and hit enter:
  `{"config":{"w":640,"h":720,"url":"https://www.livejasmin.com/en/auth/login"}}`
* it should print out a bunch of cookies

@lordmaker You can try the same steps on Windows, if you know how to do that

**Comment by lordmaker** [comment]
nope I have no idea how to open a terminal on windows, is that even possible through cmd?

**Comment by Sojiro84** [comment]
I did as you asked and it did printed out a bunch of cookie information. Anything else I need to do?

**Comment by 0xboobface** [comment]
@Sojiro84 Thank you, I just wanted to clarify, that the browser part works. The information you see, normally would be transferred to ctbrec and somehow it doesn't happen.

@lordmaker You could do it with cmd, but you would need a telnet client. For the telnet part, you can also use putty. But this is only a test, it won't fix your problem. I'm just trying to understand why it is not working.
